src/System.Threading.Thread/ref/System.Threading.Thread.csproj

```diff
@@ -9,6 +9,10 @@
     <Compile Include="System.Threading.Thread.cs" />
   </ItemGroup>
   <ItemGroup>
+    <!-- ToDo: Remove this reference once prerelease gets updated again -->
+    <ProjectReference Include="..\..\System.Runtime\ref\System.Runtime.csproj" />
```

Comment:
I don't believe you will be needing this, since you didn't touch System.Runtime and corefx versions were updated to latest this morning.